x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/LifecyclePolicy.java

```diff
@@ -203,7 +207,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         }
 
         if (phase != null) {
-            Step.StepKey afterStepKey = new Step.StepKey(phase.getName(), "pre-" + lastStepKey.getAction(), "after");
+            // The very first after step is in a phase before the hot phase so call this "new"
+            Step.StepKey afterStepKey = new Step.StepKey("new", PhaseAfterStep.NAME, PhaseAfterStep.NAME);
```

Comment:
So, for documentation purposes, our phases are now

1. new
2. hot *(optional)*
3. warm *(optional)*
4. cold *(optional)*
5. delete *(optional)*
6. completed

Reply:
yes, although we should be clear in the docs that `new` and `completed` are not real phases in the fact that you cannot add actions to them